src/pybamm/plotting/quick_plot.py

```diff
@@ -484,19 +484,19 @@ def plot(self, t, dynamic=False):
 
         Parameters
         ----------
-        t : float
-            Dimensional time (in 'time_units') at which to plot.
+        t : float or list
+            Dimensional time (in 'time_units') at which to plot. Can be a single time or a list of times.
         dynamic : bool, optional
             Determine whether to allocate space for a slider at the bottom of the plot when generating a dynamic plot.
             If True, creates a dynamic plot with a slider.
         """
 
         plt = import_optional_dependency("matplotlib.pyplot")
         gridspec = import_optional_dependency("matplotlib.gridspec")
-        cm = import_optional_dependency("matplotlib", "cm")
-        colors = import_optional_dependency("matplotlib", "colors")
 
-        t_in_seconds = t * self.time_scaling_factor
+        if isinstance(t, (int, float)):
```

Comment:
Probably easier to do it this way
```suggestion
        if not isinstance(t, list):
```